Assets/Scripts/Runtime/Enums/EventTypeCaller.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+namespace Assets.Scripts.Runtime.Enums
+{
+    public enum EventTypeCaller
+    {
+        ByTime = 1,
```

Comment:
doesnt look usefull, consider real scenarios in building a level where this could be used
more info in game manager comment